src/Elewant/FrontendBundle/ElewantFrontendBundle.php

```diff
@@ -17,6 +17,8 @@ public function boot()
          * This is probably not the place to put a custom Doctrine type...
          * (no need to look at git blame: it was me: @f_u_e_n_t_e)
          */
-        Type::addType('breed', Breed::class);
+        if (!Type::hasType('breed')) {
+            Type::addType('breed', Breed::class);
+        }
```

Comment:
You can put it in `app/config/config.yml`:

```yml
doctrine:
    dbal:
        types:
            breed:
                class: Elewant\FrontendBundle\Entity\CustomType\Breed
```

See https://symfony.com/doc/current/reference/configuration/doctrine.html 😉